docs/contribute/document.rst

```diff
@@ -20,7 +20,7 @@
 Write Document and Tutorials
 ============================
 
-We use the `Sphinx <http://sphinx-doc.org>`_ for the main documentation.
+We use the `Sphinx <https://sphinx-doc.org>`_ for the main documentation.
```

Comment:
this one seems to be not working.

Reply:
good catch, just reverted this one to http